src/06-loops.js

```diff
@@ -25,7 +31,16 @@ function rangeSum(min, max) {}
  *  rangeOdd(10, 15);
  *  //> [ 15, 13, 11 ]
  */
-function rangeOdd(min, max) {}
+function rangeOdd(min, max) {
+  let sum = [];
+  for (let i = max; i >= min; i--) {
+    if (i % 2 !== 0){
+      sum.push(i);
+    }
+  } 
+  
+  return sum;
+}
```

Comment:
You folks are solving problems to the point where it's time to think beyond getting the problem correct. In this case, I'd encourage you to think for a moment as you name variables—is this name as specific and accurate as it can be? In this case, your return value is not a sum!